html/index.html

```diff
@@ -24,13 +24,22 @@
             </div>
             <div>
                 <ul class="vertical-centered">
-                    <li><button id="button-new" class="button button--sacnite button--inverted" alt="Remove node"><i class="fa fa-file"></i><span>+</span></button></li>
+                    <li><button id="button-new" class="button button--sacnite button--inverted" alt="Remove node"><i class="icon-file"></i><span>+</span></button></li>
```

Comment:
For all these buttons, could you add a title attribute, with the same text as the alt attribute? In some browsers (like Chrome in Widnows, which I am currently using) using the title attribute makes a tooltip appear when I hover over the button for a couple of seconds, but alt doesn't.